web/packages/teleport/src/Audit/fixtures/index.ts

```diff
@@ -1625,6 +1625,25 @@ export const events = [
     uid: '4a613b84-7315-41f4-9219-1afd6b08d4da',
     user: 'alice@example.com',
   },
+  {
+    cluster_name: 'root',
+    code: 'TDB00I',
+    db_name: 'test',
+    db_protocol: 'postgres',
+    db_service: 'local',
+    db_uri: 'localhost:5432',
+    db_user: 'postgres',
+    ei: 0,
+    postgres_pid: 123,
```

Comment:
nit: Is this shown anywhere on audit page? if not, i don't think it's very useful to define an example here.

Reply:
oh, no it's not displayed on the overview/list of audit events. it's just show in an individual event.

Reply:
Removing this, I see what you mean